Decide/src/main/java/DD2480/decide/CMVHandler.java

```diff
@@ -30,9 +32,22 @@ public CMVHandler(Parameter parameters, Points[] dataPoints) {
 	}
 
 	private boolean licZero() {
```

Comment:
We should have a test included in all commits that's tests the new feature.

Reply:
Got it !